src/main.zig

```diff
@@ -315,3 +315,27 @@ test "zero" {
     try serialize(u8, std.testing.allocator, 0, &out);
     try std.testing.expectEqualSlices(u8, &[_]u8{0x80}, out.items);
 }
+
+test "access list filled" {
+    const AccessListItem = struct {
+        address: [20]u8,
+        storage_keys: [][32]u8,
+    };
+    const StrippedTxn = struct {
+        access_list: []AccessListItem,
+    };
+
+    var buf: [128]u8 = undefined;
+    const rlp = try std.fmt.hexToBytes(&buf, "f83af838f7940000000000000000000000000000000000001210e1a00000000000000000000000000000000000000000000000000000000000000203");
+    var out: StrippedTxn = StrippedTxn{ .access_list = &[0]AccessListItem{} };
```

Comment:
This could be replaced by:

```suggestion
    var out: StrippedTxn = undefined;
```

which would make sense since pre-allocations don't work, but I am waiting for input on this one.

Reply:
SGTM